Makefile

```diff
@@ -8,15 +8,20 @@ SUDO =
 BUILDTAGS   = btrfs_noversion libdm_no_deferred_remove
 BUILDFLAGS := -tags "$(BUILDTAGS)"
 
+PACKAGES := $(shell go list ./... | grep -v vendor)
+
 all: deps .gitvalidation test validate
 
-deps:
-	go get -t $(BUILDFLAGS) ./...
+deps: vndr
 	go get -u $(BUILDFLAGS) github.com/golang/lint/golint
 	go get $(BUILDFLAGS) github.com/vbatts/git-validation
 
-test:
-	@go test $(BUILDFLAGS) -cover ./...
+vndr:
+	go get -u github.com/LK4D4/vndr
+	vndr
+
+test: deps
```

Comment:
Is this strictly necessary? Slowing down the edit-test cycle is not all that attractive, and I currently tend to use `go build ./... && make .gitvalidation test validate` instead of `make all` to avoid the `deps` step.

`make all` will run `deps` anyway.

At the very least, the `vndr` target should be able to use a timestamp so that `vndr` gets rerun only when `vendor.conf` is updated.